Source/Core/Common/Debug/MemoryPatches.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+
+namespace Common::Debug
+{
+struct MemoryPatch
+{
+  enum class State : bool
```

Comment:
This can likely just be a regular enum class without the direct type specifier. By definition the identifiers are distinct and strongly typed, so this just adds verbosity for no real benefit (especially when there's no use of the underlying values given, or anything being dependent on them).